site/docs/hosting/vercel.md

```diff
@@ -80,7 +80,7 @@ if (!token) throw new Error("BOT_TOKEN is unset");
 
 const bot = new Bot(token);
 
-export default webhookCallback(bot, "std/http");
+export default webhookCallback(bot, "https");
```

Comment:
Do the same for another language, please.

Reply:
Done. Pls check.